app/javascript/packs/Search.jsx

```diff
@@ -1,9 +1,9 @@
 import { h, render } from 'preact';
 import { Search } from '../src/components/Search';
-import 'focus-visible'
+import 'focus-visible';
 
 document.addEventListener('DOMContentLoaded', () => {
   const root = document.getElementById('nav-search-form-root');
 
-  render(<Search />, root, root.firstElementChild);
+  render(<Search />, root);
```

Comment:
This is part of the upgrade. VDOM diffing works the way it does in most other libraries, e.g. React.